grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/GrpcWebTrailersExtractor.java

```diff
@@ -75,12 +76,11 @@ public HttpResponse execute(HttpClient delegate, ClientRequestContext ctx, HttpR
         final HttpResponse response = delegate.execute(ctx, req);
         final ByteBufAllocator alloc = ctx.alloc();
 
-        final ArmeriaMessageDeframerHandler handler = new ArmeriaMessageDeframerHandler(maxMessageSizeBytes);
-        final HttpDeframer<DeframedMessage> deframer = newHttpDeframer(handler, alloc, grpcWebText);
-
+        final ArmeriaMessageDeframer deframer = new ArmeriaMessageDeframer(maxMessageSizeBytes);
         final DefaultStreamMessage<HttpData> publisher = new DefaultStreamMessage<>();
-        publisher.subscribe(deframer, ctx.eventLoop());
-        deframer.subscribe(new TrailersSubscriber(ctx), ctx.eventLoop());
+        final StreamMessage<DeframedMessage> deframed =
+                new DefaultHttpDeframer<>(publisher, deframer, alloc, byteBufConverter(alloc, grpcWebText));
```

Comment:
How about renaming `DefaultHttpDeframer` to `DecodedHttpStreamMessage` or something similar?

Reply:
`DecodedHttpStreamMessage` looks good to me. 💯